pkg/storegateway/gateway_blocks_http.go

```diff
@@ -149,10 +159,9 @@ func (s *StoreGateway) BlocksHandler(w http.ResponseWriter, req *http.Request) {
 	}, blocksPageTemplate, req)
 }
 
-func formatTimeIfNotZero(t time.Time, format string) string {
-	if t.IsZero() {
+func formatTimeIfNotZero(t int64, format string) string {
```

Comment:
We should use strong types instead of "integers" for times. Perhaps we could use `*time.Time` in `richMeta`?